cafe/engine/models/data_interfaces.py

```diff
@@ -140,7 +140,7 @@ class ConfigParserDataSource(DataSource):
     def __init__(self, config_file_path, section_name):
         super(ConfigParserDataSource, self).__init__()
 
-        cafe_env_var = {key: value for key, value in os.environ.iteritems()
```

Comment:
If you want to change this to fix a python 3.X compatibility issue, please make that change in a separate PR.